### ISSUE TITLE
Faking time with Sinon on Date types fails due to constructor type

### ISSUE THREAD

**Issue by YagoQuinoy** [Atinux/schema-inspector#28]
Doing some test on my app, i need to fake time due to 'new Date()'.
```javascript
clock = sinon.useFakeTimers(new Date(2015, 11, 25).getTime()); // It's always christmas!
```
So, schema inspector fails on date type validation
```
error:  Error: Property @.dates.from: must be date, but is an instance of Date
Property @.dates.to: must be date, but is an instance of Date
```
This is because Sinon wraps Date. When it comes to validate type, it's value is [Function: Date]

```javascript
// schema-inspector.js
"date": function(element) {
   console.log(element.constructor); // Prints [Function: Date]
   return element != null && element.constructor === Date; // returns false
},
```
Using `element.constructor.name` would return an string 'Date' instead, but i dont know if it's a valid solution.

**Comment by YagoQuinoy** [comment]
Could be a nice solutions use `instanceof`?
```
// schema-inspector.js
"date": function(element) {
   return element != null && element instanceof  Date; // returns true
}
```

**Comment by Atinux** [comment]
Closed on pull #29
